src/compiler/tsbuildPublic.ts

```diff
@@ -1350,6 +1350,7 @@ namespace ts {
     }
 
     function getUpToDateStatusWorker(state: SolutionBuilderState, project: ParsedCommandLine, resolvedPath: ResolvedConfigFilePath): UpToDateStatus {
+        const force = !!state.options.force;

```

Comment:
This shouldnt be here because what of scenario when `tsc --b --force --watch`

Reply:
ahh well it still is going to build all the time i guess... it was never tested scenario.. so ....

Reply:
I would have assumed that meant "force the first time, watch after that".  If it currently means "force build on every change", then I agree (I think?  I didn't completely follow your comment) that this shouldn't cause a change in behavior (other than dropping status messages).

Reply:
yes  i assumed it would be force first time and watch after that but thats not current behaviour.. 
I think it would be better to add different status message just to be clear rather than just omitting it completely.

Reply:
Done.